src/pages/Dashboard.tsx

```
@@ -233,6 +270,19 @@ export default function Dashboard() {
+            <div className="flex flex-col sm:flex-row gap-3 mt-3">
+              <Input
+                type="text"
+                placeholder="Scope (e.g., packages/, src/) - Optional"
+                value={repoScope}
+                onChange={(e) => setRepoScope(e.target.value)}
+                className="flex-1 bg-background/50 max-w-sm"
+                onKeyPress={(e) => e.key === "Enter" && handleAnalyze()}
+              />
+            </div>
+            <div className="mt-3">
+              <ShortcutHint />
```

ESLint

[error] 'ShortcutHint' is not defined.

(react/jsx-no-undef)